CHANGELOG.md

```diff
@@ -4,6 +4,117 @@
+
+With this release, we are officially moving TimescaleDB 2.0 to GA, 
+concluding several release candidates.
+
+TimescaleDB 2.0 adds the much-anticipated support for distributed 
+hypertables (multi-node TimescaleDB), as well as new features and 
+enhancements to core functionality to give users better clarity and 
+more control and flexibility over their data.
+
+Multi-node architecture:  In particular, with TimescaleDB 2.0, users 
+can now create distributed hypertables across multiple instances of 
+TimescaleDB, configured so that one instance serves as an access node 
+and multiple others as data nodes. All queries for a distributed 
+hypertable are issued to the access node, but inserted data and queries
+are pushed down across data nodes for greater scale and performance.
+
+Multi-node TimescaleDB can be self managed or, for easier operation, 
+launched within Timescale's fully-managed cloud services.
+
+This release also adds:
+
+* Support for user-defined actions, allowing users to define, 
+  customize, and schedule automated tasks, which can be run by the 
+  built-in jobs scheduling framework now exposed to users.
+* Significant changes to continuous aggregates, which now separate the 
+  view creation from the policy.  Users can now refresh individual 
+  regions of the continuous aggregate materialized view, or schedule 
+  automated refreshing via  policy.
+* Redesigned informational views, including new (and more general) 
+  views for information about hypertable's dimensions and chunks, 
+  policies and user-defined actions, as well as support for multi-node 
+  TimescaleDB.
+* Moving all formerly enterprise features into our Community Edition, 
+  and updating Timescale License, which now provides additional (more 
+  permissive) rights to users and developers.
+
+Some of the changes above (e.g., continuous aggregates, updated 
+informational views) do introduce breaking changes to APIs and are not 
+backwards compatible. While the update scripts in TimescaleDB 2.0 will 
+upgrade databases running TimescaleDB 1.x automatically, some of these 
+API and feature changes may require changes to clients and/or upstream 
+scripts that rely on the previous APIs.  Before upgrading, we recommend
+reviewing upgrade documentation at docs.timescale.com for more details.
+
+**Major Features**
+
+TimescaleDB 2.0 moves the following major features to GA:
+* #1923 Add support for distributed hypertables
+* #2006 Add support for user-defined actions
+* #2125 #2221 Improve Continuous Aggregate API
+* #2084 #2089 #2098 #2417 Redesign informational views
+* #2435 Move enterprise features to community
+* #2437 Update Timescale License
+
+**Previous Release Candidates**
+
+* #2702 Release Candidate 4 (December 2, 2020)
+* #2637 Release Candidate 3 (November 12, 2020)
+* #2554 Release Candidate 2 (October 20, 2020)
+* #2478 Release Candidate 1 (October 1, 2020)
+
+**Minor Features**
+
+Since the last release candidate 4, it includes several minor 
```

Comment:
I suggest the following reformulation since "it" refers to the current release, which is not mentioned in the sentence:

```suggestion
Since the last release candidate 4, there are several minor 
```